Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -369,24 +371,579 @@ public static void CreateRealRunSetFromDynamic(RunsetExecutor runsetExecutor, Ad
+                            mailOperation.RunAt = RunSetActionBase.eRunAt.ExecutionEnd;
+                        }                        
+
+                        if (runsetOperationConfigMail.MailSettings.EmailMethod != null)
+                        {
+                            if (runsetOperationConfigMail.MailSettings.EmailMethod == SendMailSettings.eEmailMethod.OUTLOOK)
+                            {
+                                mailOperation.Email.EmailMethod = GingerCore.GeneralLib.Email.eEmailMethod.OUTLOOK;
+                            }
+                            else
+                            {
+                                mailOperation.Email.EmailMethod = GingerCore.GeneralLib.Email.eEmailMethod.SMTP;
+                                if (runsetOperationConfigMail.MailSettings.SmtpDetails != null)
+                                {
+                                    if (runsetOperationConfigMail.MailSettings.SmtpDetails.Server != null)
+                                    {
+                                        mailOperation.Email.SMTPMailHost = runsetOperationConfigMail.MailSettings.SmtpDetails.Server;
+                                    }
+                                    if (runsetOperationConfigMail.MailSettings.SmtpDetails.Port != null)
+                                    {
+                                        mailOperation.Email.SMTPPort = int.Parse(runsetOperationConfigMail.MailSettings.SmtpDetails.Port);
+                                    }
+                                    if (runsetOperationConfigMail.MailSettings.SmtpDetails.EnableSSL != null)
+                                    {
+                                        mailOperation.Email.EnableSSL = (bool)runsetOperationConfigMail.MailSettings.SmtpDetails.EnableSSL;
+                                    }
+                                    if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.SmtpDetails.User) == false)
+                                    {
+                                        mailOperation.Email.ConfigureCredential = true;
+                                        mailOperation.Email.SMTPUser = runsetOperationConfigMail.MailSettings.SmtpDetails.User;
+                                        mailOperation.Email.SMTPPass = runsetOperationConfigMail.MailSettings.SmtpDetails.Password;
+                                    }
+                                }
+                            }
+                        }
+                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.MailFrom) == false)
+                        {
+                            mailOperation.MailFrom = runsetOperationConfigMail.MailSettings.MailFrom;
+                            mailOperation.Email.MailFrom = runsetOperationConfigMail.MailSettings.MailFrom;
+                        }
+                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.MailTo) == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)